src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/GEOS_LandAssimGridComp.F90

```diff
@@ -2375,23 +2377,34 @@ subroutine CALC_LAND_TB(gc, import, export, clock, rc)
     
     allocate(TB_h_tmp(N_catl), TB_v_tmp(N_catl))
     
-    if (.not. incl_atm_terms) then
+    select case (RTM_ID)
+       
+    case(2,4)
+
        allocate(dummy_real(N_catl))                   ! allocate needed for GNU compiler
+
        call mwRTM_get_Tb(                         &
             N_catl, freq, inc_angle, mwRTM_param, &   
-            dummy_real,                           &   ! intent(in), "elev", not used as long as "incl_atm_terms=.false."
+            dummy_real,                           &   ! intent(in), "elev", not used as long as RTM_ID=4 (formerly "incl_atm_terms=.false.")
             LAI,                                  &   
             sfmc_mwRTM,                           &   
             tsoil_mwRTM,                          &   
             SWE,                                  &   
             dummy_real,                           &   ! intent(in), "Tair", not used as long as "incl_atm_terms=.false." 
-            incl_atm_terms,                       &   
-            'mironov',                       &
+            RTM_ID,                               &
             Tb_h_tmp, Tb_v_tmp )                      ! intent(out) 'TB_LAND_1410MHZ_40DEG_HPOL',  'TB_LAND_1410MHZ_40DEG_VPOL'
        deallocate(dummy_real) 
-    else
-       _ASSERT(.false., "top-of-atmosphere Tb calculation not yet implemented (incl_atm_terms=.true.)")
-    end if
+
+    case(1,3)
+       
+       _ASSERT(.false., "top-of-atmosphere Tb calculation (requested per RTM_ID) not yet implemented")
+       
+    case default
+
+       err_msg = 'unknown RTM_ID (during CALC_LAND_TB)'
+       call ldas_abort(LDAS_GENERIC_ERROR, Iam, err_msg)
```

Comment:
need to declare ldas_abort, LDAS_GENERIC_ERROR in this file

Reply:
should be fixed now.  let's see if the build test works.